### DISCUSSION TITLE
Welcome to VaadinNav Discussions!

### DISCUSSION THREAD

**Discussion by Peppe** [Peppe/ApolloNav#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by macrotea** [comment]
hi，I want to understand how this menu can be expanded by clicking on the text of the menu item without clicking the icon on the far right. Can you tell me how to achieve this, trouble?
